### ISSUE TITLE
Make the "episode listened until the end" indicator screen reader accessible

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#858]
> No issues, I love the app. Just wondering if I’m blind and can’t see an indicator for whether or not a podcast has played. If not I would like to suggest having an indication that a track has been listened too. Thank you

**Comment by mitchdowney** [comment]
This should be in the next release 4.3.6 https://github.com/podverse/podverse-rn/commit/854790cf940b4d7505e8e953cfac5f1f30d7eee9
